core/commands/keystore.go

```diff
@@ -202,6 +203,61 @@ var KeyListCmd = &cmds.Command{
 	Type: KeyOutputList{},
 }
 
+var KeyRmCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Remove a keypair",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("name", true, false, "name of key to remove"),
```

Comment:
This command should allow for StdIn handling and multiple argument handling. Then it would be possible to remove multiple keys at the same time easily.